### DISCUSSION TITLE
Hi! Is there a cheat that lets me remove buildings/ move them?

### DISCUSSION THREAD

**Discussion by Boji2005** [RedShyGuy/Vapecord-ACNL-Plugin#80] [Q&A] [answered]
Hello! Does a cheat like that exist? I did find a video where a person used "Real time building placer". But it seems like that person Is using an older version of this plugin(or not this pluggin at all) so I'm asking If someone knows how to help me

**Comment by RedShyGuy** [comment, marked as answer]
Yes there is one, it's called "Building Modifier" and it's inside the quick menu cheat (in Misc Codes)

**Reply by Boji2005** [reply]
Thanks again! You're awesome!
